interactive-movie-app/src/app/components/credits/credits.component.html

```diff
@@ -1 +1,143 @@
-<p>credits works!</p>
+<!--<!DOCTYPE html>
```

Comment:
doctype und head wird für ein component in Angular nicht benötigt, das läuft alles über Angular